subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsTask.kt

```diff
@@ -229,16 +229,17 @@ abstract class InstrumentationTestsTask @Inject constructor(
 
     private fun createReportFactory(): Report.Factory {
         val reportViewerConfig = reportViewerConfig.orNull
-        val factories = mutableMapOf<Class<out Report.Factory.Config>, Report.Factory>()
+        val factories = mutableMapOf<String, Report.Factory>()
```

Comment:
Why did you weaken a key? A string is more error-prone, can write any value.

Reply:
New type inference in 1.4 breaks this and i don't know how to fix it properly